.github/format-check.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -ex
+
+if [ "$GITHUB_EVENT_NAME" == "pull_request" ]; then 
+    for f in $(git diff --name-only origin/$GITHUB_BASE_REF | grep -E "(vita3k|tools/gen-modules|tools/native-tool)/.*(.cpp|.h)" ); do
```

Comment:
You could use ```git diff --name-only origin/$GITHUB_BASE_REF HEAD 'vita3k/**.cpp' 'vita3k/**.h' 'tools/**.cpp' 'tools/**.h'``` to make this line like 10+ times faster, but I suppose it doesn't matter much. In theory you could also add `--diff-filter=AM` (only show `added` or `modified` files, could be extended to`R` and `C` (renamed/copied)) so you don't need to check if the file exists.
But otherwise it looks ok.

Reply:
✅